public/assets/js/scripts.js

```diff
@@ -118,4 +127,34 @@ $(document).ready(function(){
       }).on('hidden.bs.collapse', function(){
       $(this).find('.faq-icon, .faq-option-category-icon').text((text) => text === '+' ? '-' : '+');     
   });
+
+  //function to format number input values to currency like format
+  $(".investment-value-input").on('keyup', function(){
+    var number = parseInt($(this).val().replace(/\D/g,''),10);
+    number = (isNaN(number) ? 0 : number);
+    $(this).val(number.toLocaleString());
+  });
+
+  const rangeSliderDiv = document.querySelectorAll('.custom-range-div');
+  //update the input & label value when slider is moved
+  Array.prototype.forEach.call(rangeSliderDiv,(slider)=>{ 
+    slider.querySelector('.custom-range').addEventListener('input', (event)=>{
+      var inputValue = removeCommasAndMakeNumber(event.target.value)
+      slider.querySelector('label').innerHTML = 'NGN ' + formatToCurrency(inputValue);
+      slider.querySelector('.form-control').value = formatToCurrency(inputValue);
+    });
+  });
+
+  // update the slider thumb location & label when input value changes
+  Array.prototype.forEach.call(rangeSliderDiv,(input)=>{ 
```

Comment:
@tobieeniafe, Instead of looping through the inputs here, can you create a separate function for the event handler (processInputValue) and call each input separately to handle the maximum value. Here is the rough draft of the implementation (feel free to modify it to your taste to make it work flawlessly);

1. Create a custom function that does the same this as this but without hooking it with an event handler.
2. For the first input `Initial Payment amount textbox`, add an event handler, and check if the value exceeds the proposed maximum amount (30,000,000). If it does, reset to 30,0000. The created function in 1, will be called after the verification
3. Repeat step 2 for the Periodic Investment Amount.

Please let me know if any part of the implementation is uncleared. Thank you and Nice work.